run_benchmarks

```diff
@@ -445,6 +445,9 @@ def main():
     parser.add_argument('--benchmarks', type=str,
                         help='comma-separated list of benchmarks to run ' +
                              '(regular expressions are supported)')
+    parser.add_argument('type', type=str,
+                        choices={b['name'].split('-', 1)[0] for b in benchmarks},
+                        help='type of benchmark to run')
```

Comment:
Since we're at it, I would kill `--benchmarks` and make `"type"` an n-arg which supports prefix matching and regexes, i.e:

```shell
$ run_benchmarks -D 10 http '.*tornado.*'
```